modules/tenant-management/src/Volo.Abp.TenantManagement.Domain.Shared/Volo/Abp/TenantManagement/TenantConsts.cs

```diff
@@ -6,4 +6,8 @@ public static class TenantConsts
     /// Default value: 64
     /// </summary>
     public static int MaxNameLength { get; set; } = 64;
+
+    public const int MaxPasswordLength = 128; 
+    
+    public const int AdminEmailAddress = 256; 
```

Comment:
rename to `MaxAdminEmailLength` is better. and why here use `const` not `static`?
or can we reuse `AbpUserConsts.MaxEmailLength` ?:https://github.com/abpframework/abp/blob/48c02d787503356fcc56b9234cb7f11e23a5a8be/modules/users/src/Volo.Abp.Users.Domain.Shared/Volo/Abp/Users/AbpUserConsts.cs#L23
just like:
https://github.com/abpframework/abp/blob/48c02d787503356fcc56b9234cb7f11e23a5a8be/modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/IdentityUserConsts.cs#L15

Reply:
1. I need to the const value because it should be changed
2. Attributes accept just const value

Reply:
const value will not allow developer to change outside of module.
use `DynamicStringLengthAttibute`, see: https://github.com/abpframework/abp/blob/5ef9707adc458851c3bfe0c03063811c2596e8c5/modules/tenant-management/src/Volo.Abp.TenantManagement.Web/Pages/TenantManagement/Tenants/CreateModal.cshtml.cs#L40